inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -306,7 +306,11 @@ class XmlSerializer : public ngraph::AttributeVisitor {
                 input_descriptions_on_adapter(a->get(), parameter_mapping, result_mapping, port_map);
             } else if (const auto& a = ngraph::as_type<ngraph::AttributeAdapter<std::vector<std::shared_ptr
                                 <ngraph::op::util::SubGraphOp::OutputDescription>>>>(&adapter)) {
-                output_descriptions_on_adapter(a->get(), parameter_mapping, result_mapping, port_map);
+                uint32_t op_input_count = 0;
```

Comment:
Please add necessary tests.

Reply:
test validating serialized external_port_id is added.